controllers/telemetry/logpipeline_controller.go

```diff
@@ -60,16 +61,18 @@ func (r *LogPipelineReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 func (r *LogPipelineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&telemetryv1alpha1.LogPipeline{}).
+		Owns(&corev1.Secret{}).
```

Comment:
Replace the `Owns` calls with `Watches` as it is done [here](https://github.com/kyma-project/telemetry-manager/blob/main/controllers/telemetry/tracepipeline_controller.go#L60). This has the advantage that also changes on owned resources without the controller flag trigger a reconciliation.